connector-raw/scripts/create_schema.sql

```diff
@@ -22,48 +22,21 @@ CREATE TABLE pubkey (
     pubkey VARCHAR(44) NOT NULL UNIQUE
 );
 
--- Returns a pubkey_id for a pubkey, by getting it from the table or inserting it.
--- Getting this fully correct is complex, see:
--- https://stackoverflow.com/questions/15939902/is-select-or-insert-in-a-function-prone-to-race-conditions/15950324
--- and currently this function assumes there are no deletions in the pubkey table!
-CREATE OR REPLACE FUNCTION map_pubkey(_pubkey varchar(44), OUT _pubkey_id bigint)
-  LANGUAGE plpgsql AS
-$func$
-BEGIN
-   LOOP
-      SELECT pubkey_id
-      FROM   pubkey
-      WHERE  pubkey = _pubkey
-      INTO   _pubkey_id;
-
-      EXIT WHEN FOUND;
-
-      INSERT INTO pubkey AS t
-      (pubkey) VALUES (_pubkey)
-      ON     CONFLICT (pubkey) DO NOTHING
-      RETURNING t.pubkey_id
-      INTO   _pubkey_id;
-
-      EXIT WHEN FOUND;
-   END LOOP;
-END
-$func$;
-
 -- The table storing account writes, keeping only the newest write_version per slot
 CREATE TABLE account_write (
-    pubkey_id BIGINT NOT NULL REFERENCES pubkey,
+    pubkey VARCHAR NOT NULL,
     slot BIGINT NOT NULL,
     write_version BIGINT NOT NULL,
+    owner VARCHAR NOT NULL,
     is_selected BOOL NOT NULL,
-    owner_id BIGINT REFERENCES pubkey,
     lamports BIGINT NOT NULL,
     executable BOOL NOT NULL,
     rent_epoch BIGINT NOT NULL,
     data BYTEA,
-    PRIMARY KEY (pubkey_id, slot, write_version)
+    PRIMARY KEY (pubkey, slot)
 );
-CREATE INDEX account_write_searchkey on account_write(pubkey_id, slot DESC, write_version DESC);
-CREATE INDEX account_write_pubkey_id_idx on account_write(pubkey_id);
+CREATE INDEX account_write_searchkey on account_write(pubkey, slot DESC, write_version DESC);
+CREATE INDEX account_write_pubkey_id_idx on account_write(pubkey);
```

Comment:
when you create new indexes here you can add them here in this file so we have some documetnation. I think (owner,slot) will be most important. 

later we'll need to index the `data` column using substring indexes if we can.